eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/processor/inf/AbstractEventProcessor.java

```diff
@@ -125,8 +134,10 @@ protected String getTargetMesh(String consumerGroup, List<SubscriptionItem> subs
                     JsonUtils.deserialize(topicMetadataJson, ConsumerGroupMetadata.class);
             Map<String, ConsumerGroupTopicMetadata> consumerGroupTopicMetadataMap =
                     Optional.ofNullable(consumerGroupMetadata)
-                        .map(ConsumerGroupMetadata::getConsumerGroupTopicMetadataMap)
-                        .orElse(Maps.newConcurrentMap());
+                            .map(ConsumerGroupMetadata::getConsumerGroupTopicMetadataMap)
+                            .orElseGet(() -> {
+                                return Maps.newConcurrentMap();
```

Comment:
I think it can be replaced here with .orElseGet(Maps::newConcurrentMap).

Reply:
Thanks.I've fixed it.